src/index.scss

```diff
@@ -270,7 +295,8 @@ $select-input-height: 40px; // Bootstrap input height
 // MENU STYLE ==================================================================
 
 .xo-menu, .xo-sub-menu {
-  background-color: $gray;
+  // background: linear-gradient(to top, $brand-primary, $brand-secondary);
+  background: $brand-menu;
   color: white;
```

Comment:
If the `background` is a variable, it would make sense for the `color` to be too.